SEO updates for the PGCE page (#4426)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/what-is-a-pgce.md

    link_url: /non-uk-teachers/non-uk-qualifications
---

> After gaining a degree, you may want to go on to complete a postgraduate certificate (PG certificate) or diploma (PG diploma).

Reviewer: One for policy on the phrasing re PG cert and PG diploma!

Reviewer: Also - I find the `After gaining a degree` slightly odd - I was a bit confused about this starting the page as I wasn't sure how this related to teaching. I think the sentence feels a bit weird because it is talking about general postgrad qualifications rather than teaching - like you are just getting a postgrad qualification for the fun of it?

Author: ```suggestion
If you have a degree and are interested in gaining a qualification in education, you may want to do a postgraduate certificate in education (PGCE).
```

Author: Have reworded this, if it makes it more confusing (which I agree!) then it's defo not worth including as they're fringe keywords anyway (as in, general 'useful to have' keywords rather than exact keywords)

> You can get a postgraduate certificate in education (PGCE) through teacher training.

Reviewer: ```suggestion
You can get a postgraduate certificate in education through teacher training.
```
Do we need the acronym again here? It's in the sentence before? Or should we just say PGCE in this sentence rather than writing it out in full again?

You do not need a PGCE to be a qualified teacher in England, but it can help you develop your teaching skills and knowledge.

## The qualifications you need to teach in England

You need qualified teacher status (QTS) to teach in most primary, secondary and special schools in England. You can get QTS through undergraduate or postgraduate teacher training.

[Find out more about qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).

> Many teacher training courses also offer a postgraduate certificate, such as a PGCE, but you do not need one to teach.

Author: ```suggestion
Many teacher training courses also offer a postgraduate qualification, such as a PGCE, but you do not need one to teach.
```

> Many teacher training courses also offer a postgraduate certificate, such as a PGCE, but you do not need one to teach.

Author: Changing back as it's not actually a keyword

Getting a PGCE without QTS means you will not be qualified to teach in most primary, secondary and special schools in England, although some schools will employ teachers without QTS.

If you get a PGCE without QTS you will not be entitled to the same benefits during your training and career as QTS with PGCE. For example, better pay and support when you start teaching.

## Benefits of having QTS with PGCE

> PGCE can give you a better understanding of teaching practices, educational research and theory.

Author: ```suggestion
A PGCE can give you a better understanding of teaching practices, educational research and theory.
```
